adobe-apiplatform/user-sync.py
# remove six pkg references

## Summary
* Remove all usage of "six" in the UST codebase.
-  changes are 
    - six.iteritems(), we can use items() on the collection
    - six.iterkeys() can be replaced by calling keys() on the collection
    - six.itervalues() can be replaced by calling values() on the collection

## Testing Steps
*  Run unit test using "pytest".
*  Check all application sync process by running application on test mode.



Fixes #760
